Bottom: 5d654240ddfbc076b509ebb099b7d030ca9e8e15
Top:    918829e1c0d9b88b6f2c91c3fa40e920c207a83b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:59:08 +0000

Revert "Remaining (bug?) from another patch.."

This reverts commit a349202698b3ef11a2a886b300d55e201e15bd43.

NB: Original function had this bug before we moved it


---

diff --git a/src/misc.c b/src/misc.c
index 7e9f635..1729bbd 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -397,7 +397,7 @@ calc_arc_from_points_and_included_angle (PointType *p1, PointType *p2, Angle inc
   hx = p1->X + (p2->X - p1->X) / 2;
   hy = p1->Y + (p2->Y - p1->Y) / 2;
 
-  p_to_h_dist = hypot (p2->X - p1->Y, p2->Y - p1->Y) / 2.;
+  p_to_h_dist = hypot (p2->X - p1->X, p2->Y - p1->Y) / 2.;
   c_to_h_dist = p_to_h_dist / tan (TO_RADIANS (included_angle) / 2.);
 
   unit_hcx = (double)-(hy - p1->Y) / p_to_h_dist;
